Sources/LiveKit/Core/Room+EngineDelegate.swift

```diff
@@ -207,13 +207,26 @@ extension Room {
             return
         }
 
-        let segments = packet.segments.map { $0.toLKType() }
+        let segments = packet.segments.map { segment in
+            TranscriptionSegment(id: segment.id,
+                                 text: segment.text,
+                                 language: segment.language,
+                                 firstReceivedTime: _state.transcriptionReceivedTimes[segment.id] ?? Date(),
```

Comment:
are you expecting both `startTime` and `firstReceivedTime` to have separate purposes? 
Otherwise we could simply override `startTime`  with `firstReceivedTime`?

Reply:
I don't know what startTime and endTime are intended for but I assumed they'd be in a track-local timekeeping system? They don't have a Date - it's UInt64. When we do eventually add support for them from the serve, would that cause a breaking change if we'd overloaded them already in the clients?